src/lib-tempo/src/component/ComponentStage.cpp

```diff
@@ -111,6 +111,15 @@ bool ComponentStage::existstTile(glm::ivec2 position)
 		return false;
 }
 
+bool ComponentStage::isNavigable(glm::ivec2 position)
+{
+	if (existstTile(position))
+	{
+		return getHeight(position) < 5;
```

Comment:
Not sure if this is what we want - isn't the idea that its only if the delta between two adjacent tiles is too great then you can't move from one to another, rather than there being some absolute cut off of "this is high, so is a wall"